scripts/zones/Maze_of_Shakhrami/mobs/Argus.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- Area: Maze of Shakhrami
+--  MOB: Argus (NM)
+-----------------------------------
+
+require("scripts/zones/Maze_of_Shakhrami/MobIDs");
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob,killer,ally)
+    local whichNM = math.random(0,19);
+
+    if (whichNM < 10) then
+        SetRespawnTime(Argus, 64800, 108000); -- 18-30 hours
+        DeterMob(Leech_King, true);
```

Comment:
Come to think of it, if you're deterring the mob in the zone.lua on boot, you don't need to deter the other one again on death. Since Leech King is already deterred if you defeat Argus.